evadb/udfs/chatgpt.py

```diff
@@ -95,30 +96,45 @@ def completion_with_backoff(**kwargs):
             len(openai.api_key) != 0
         ), "Please set your OpenAI API key in evadb.yml file (third_party, open_api_key) or environment variable (OPENAI_KEY)"
 
-        prompts = text_df[text_df.columns[0]]
         queries = text_df[text_df.columns[0]]
```

Comment:
@Chitti-Ankith Can you add comments with two examples of how this works for different types of inputs -- what would be `prompts` and `queries` etc.

For these two examples `ChatGPT(prompt, query)` and `ChatGPT(query)`. We could add this in the UDF doc string as well.